src/common/active_window.rs

```diff
@@ -4,6 +4,7 @@ use super::window_position::WindowPosition;
 pub struct ActiveWindow {
     pub title: String,
     pub process_name: String,
+    pub app_name: Option<String>,
```

Comment:
Let's use an empty string instead of an `Option` to match other fields. The way the library works right now is you either get all fields or it fails (which might change in the future but it keeps it simple for now). And returning an empty string for platforms where implementation is missing for now is fine.

Reply:
changed in 4016baa